scripts/export-monitoring.bat

```diff
@@ -3,6 +3,13 @@ setlocal enabledelayedexpansion
 
 SET scriptpath=%~dp0
 SET diagpath=%scriptpath:~0,-1%
+SET libpath=%diagpath%\lib\NUL
```

Comment:
Do these all need the `\NUL` terminator? You're consistently only doing it in the `.bat` files, but I'm just not used to seeing it and wanted to call out in case it was leftover from something.

Reply:
It's a Windows thing. When checking on stackoverflow there appeared to be some controversy over whether the NUL was needed. Evidently in some cases Windows won't check for a folder, only a file and this forces the directory to a file so it always works. It does work without it in my testing but I left it in there in case it's tied to some Windows version or update.